README.md

```diff
@@ -34,7 +34,9 @@ $ wget https://downloads.raspberrypi.org/raspbian_lite/images/raspbian_lite-2019
 Extract the raw Raspberry Pi disk image:
 
 ```bash
-$ unzip 2019-06-20-raspbian-buster-lite.zip && cd ..
+$ unzip 2019-06-20-raspbian-buster-lite.zip
+$ MC_IMGFILE=$(ls *raspbian-buster*.img)
```

Comment:
`MC_IMGFILE` seems a bit cryptic to me. Would name it `INPUT_DISK_IMAGE` or `INPUT_RAW_DISK_IMAGE`

Reply:
Went for option nr 2